### PR TITLE
Install script

### PR DESCRIPTION
**Pull request by bit-man**
Hi,

Created an install script. Hope you like it and approve the pull request.

Thanks!

### CODE REVIEW

**Comment by bit-man** [comment]
@f3flight could you take a look at this?
Thanks!

**Comment by f3flight** [comment]
@bit-man sure, will check! I also will roll out an update soon which will allow automatic hostname confirm! so we will be able to ignore these "Confirm Your Hostname Now" emails - service will do that

File: install [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    Check noip status and log : sudo systemctl status noip
+    Enable noip to start on boot : sudo systemctl enable noip
+
+END
+}
+
+function uninstall() {
+    rootCheck
+    rm ${folder[bin]}/noip
+    rm -rf ${folder[cfg]}
```
Comment (f3flight) [inline install]: Please do not remove user configuration files on uninstall, unless they match default files exactly

File: install [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+function secureConfig() {
+  chown -R `callerUserId` ${folder[cfg]}
+  chmod -R 600 ${folder[cfg]}/*.netrc
+}
+
+function copyFiles() {
+  cp usr/sbin/* ${folder[bin]}
+  cp -R etc/noip/* ${folder[cfg]}
```
Comment (f3flight) [inline install]: I've just lost my noip.conf =) please do not rewrite existing config files here

**Review by f3flight — CHANGES_REQUESTED** [review]
I've added comments, please address

**Comment by f3flight** [comment]
@bit-man check out the updated version. Also, please fix the installer =)

**Comment by bit-man** [comment]
@f3flight Installer fixed !!

**Pull request merged**